lib/repository/base.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Repository
+  class Base
+    class << self
+      delegate :all,
```

Comment:
to make it work it should be the name of the method. 
```
delegate :all -> delegate :find_all
```

Reply:
Oh I did not see it (ctrl+z misstake :D )